### PR TITLE
Made more use of lodash functions, and added it as a dependency (it was missing)

### PR DESCRIPTION
**Pull request by Page-**
…as missing)

[Review on Reviewable](https://reviewable.io/reviews/resin-io/docker-storage-gc/1)

### CODE REVIEW

File: lib/index.coffee [hunk 1]
```diff
@@ -68,33 +68,21 @@ compare = (a, b) ->
...
-		minArrayIndex = null
-		for arr, index in arrs when arr.length isnt 0
-			if min is null or comp(arr[0], min) < 0
-				min = arr[0]
-				minArrayIndex = index
-		ret.push(min)
-		arrs[minArrayIndex].shift()
-
-	return ret
+exports.merge = merge = (arrs, comp = 0) ->
```
Comment (petrosagg) [inline lib/index.coffee]: `comp` needs to be a comparison function. I meant to pass the compare function defined above which compares based on two fields but missed it.

Comment (Page-) [inline lib/index.coffee]: Ok, I'll change this bit to work with that in mind

Comment (petrosagg) [inline lib/index.coffee]: Thanks

Comment (Page-) [inline lib/index.coffee]: Done

**Comment by CameronDiver** [comment]
@Page- what is the status of this? I assume it's probably worth closing given the changes to the codebase since this was opened?

**Pull request closed without merging**